scripts/localize.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Check if environment variables have been defined for our team ID, AAD App ID, and AAD App Secret for automated workflows before requesting them interactively
+if [ -z $TDBUILD_TEAM_ID ]; then
```

Comment:
If I understand this correctly, the secrets will exist on github / the build server, and developers of FluentUI-Apple won't need them for setup on local machines?

Reply:
If you are manually running the localization workflow on your machine and you don't have the env variables, this will prompt you interactively to provide each of the variables in the command line. I guess you _could_ set the env variables locally, and it would work exactly as it does in CI.

Reply:
But maybe to more correctly answer your question, I don't anticipate developers running the localization pipeline locally unless they are actively changing how the pipeline works. The whole localization story should be abstracted from developers by this pipeline.